src/apolo_app_types/protocols/common/k8s.py

```diff
@@ -33,8 +38,21 @@ class Env(AbstractAppFieldType):
             description="K8S container env var.",
         ).as_json_schema_extra(),
     )
-    name: str
-    value: str | int | ApoloSecret | None
+    name: str = Field(
+        ...,
```

Comment:
what's the diff between using `default=` and `...` in this context?

Reply:
When we put Ellipsis (...) it means that it is required